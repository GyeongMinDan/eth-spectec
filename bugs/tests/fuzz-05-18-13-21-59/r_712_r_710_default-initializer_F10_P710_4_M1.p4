header H {
      }
       control C(out bit<32> x)() {
        H[6s0] stack;
        apply {
         H h0 = ...;
         stack = {
    h0, ... };
       }
      }
