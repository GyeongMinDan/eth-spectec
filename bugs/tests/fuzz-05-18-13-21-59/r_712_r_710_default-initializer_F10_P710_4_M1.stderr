likely_hits/r_712_r_710_default-initializer_F10_P710_4_M1.p4(3): [--Wwarn=unused] warning: Control C is not used; removing
       control C(out bit<32> x)() {
               ^
[--Wwarn=missing] warning: Program does not contain a `main' module
